pkgs/development/tools/glamoroustoolkit/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, lib, makeWrapper, fetchzip, pkgs, xorg }:
```

Comment:
Never use pkgs or xorg

Reply:
Hi @AndersonTorres ,
Thanks for your feedback.  I've modified it as you suggested.
Cheers,
Alistair